model/feature_flags.go

```diff
@@ -60,6 +60,9 @@ type FeatureFlags struct {
 
 	// A/B test for the add members to channel button, possible values = ("top", "bottom")
 	AddMembersToChannel string
+
+	// Enable different treatments for first time users, possible values = ("none", "toggle")
```

Comment:
This comment may be a copy/paste error? Enable treatments for invite to team?

Reply:
yep!